metadata-analysis/Metadata-Analysis.Rmd

```diff
@@ -217,13 +389,160 @@ Eva did not begin tracking downloads until December 29, 2022.
 
 counts_by_week %>%
   filter(str_detect(Event, "Download")) %>%
+  filter(!str_detect(Event, "DEMO MODE")) %>%
+  ggplot(aes(x = WeekEnding, y = n, group = Event, color = Event, label = n)) +
+  geom_line() +
+  geom_point() +
+  geom_text(nudge_x = .5, color = "gray15", check_overlap = TRUE) +
+  scale_color_brewer(palette = "Dark2") +
+  scale_x_date(date_breaks = "1 week", date_labels = "%m-%d") +
+  labs(title = "Weekly Eva Downloads",
+       x = "Week Ending",
+       y = "",
+       caption = "Download data not collected until Dec 29, 2022.") +
+  theme_minimal() +
+  theme(legend.position = "bottom") +
+  guides(color = guide_legend(ncol = 2))
+
+
+```
+
+
+```{r fig.width=10, fig.height=5}
+
+counts_category_week %>%
```

Comment:
this also doesn't show correctly for me, same thing as above I think!

Reply:
fixed